docs/data/data-grid/components/components.md

```diff
@@ -195,19 +197,69 @@ declare module '@mui/x-data-grid' {
 
 <DataGrid
   slots={{
-    // custom component passed to the `toolbar` slot
```

Comment:
I removed them because codeblocks do not work properly when there's a backtick inside them.
I've opened a PR with a fix https://github.com/mui/material-ui/pull/37950. Once merged, I can add them back ;)